e2e-tests/cypress/support/commandsGeneral.ts

```diff
@@ -0,0 +1,17 @@
+Cypress.Commands.add('setResolution', size => {
+  if (Cypress._.isArray(size)) {
+    cy.viewport(size[0], size[1]);
+  } else {
+    cy.viewport(size);
```

Comment:
Returning `cy.viewport(size);` makes the function chainable (see other comment in `index.d.ts` file)